acceptance/util_test.go

```diff
@@ -99,14 +110,17 @@ func farmer(t *testing.T, prefix string) *terrafarm.Farmer {
 		t.Fatalf("farmer prefix must match regex %s", prefixRE)
 	}
 	f := &terrafarm.Farmer{
-		Output:    os.Stderr,
-		Cwd:       *flagCwd,
-		LogDir:    logDir,
-		KeyName:   *flagKeyName,
-		Stores:    stores,
-		Prefix:    prefix,
-		StateFile: prefix + ".tfstate",
-		AddVars:   make(map[string]string),
+		Output:  os.Stderr,
+		Cwd:     *flagCwd,
+		LogDir:  logDir,
+		KeyName: *flagKeyName,
+		Stores:  stores,
+		Prefix:  prefix,
+		// Prepend Terraform resource names and state file name with date/time to
+		// allow concurrent runs of the same test.
+		StateFile:            timeutil.Now().Format("20060102-150405-") + prefix + ".tfstate",
+		AddVars:              make(map[string]string),
+		KeepClusterAfterTest: *flagTFKeepCluster,
```

Comment:
`localcluster` has a flag

```go
var waitOnStop = flag.Bool("w", false, "wait for the user to interrupt before tearing down the cluster")
```

I don't know whether it makes sense to use the same flag for both (the behaviour isn't quite the same: with the local cluster, you still tear it down after CTRL+C whereas here you would leave everything running), but wanted to mention it.